libnmstate/nm/nmclient.py

```diff
@@ -92,8 +92,7 @@ def actions_exists(self):
 
     def run(self, timeout):
         if timeout is None:
-            self._mainloop.run()
-            return _MainLoop.SUCCESS
+            timeout = 3600
```

Comment:
This does not seem to be a sane default. Nobody will want to wait 60 minutes IMHO. How about 20 seconds as used in netapplier.

Reply:
it is not a default, it is the alternative of "no timeout". Previously it was indefinitely, now it is limited to "long time".
It should not be used unless in a debug mode or something.

Reply:
indefinitely is wrong, too, I agree. If there needs to be an insanely high timeout, then it could be set to a higher value in a debug build. But it should also not block for 60 minutes by accident. Also if it should not be used, then it can be a sane value like 20 seconds anyhow, so there is no reason to set it that high. :-)

Reply:
It was originally copied from the anisble roles, so this is a translation of that indefinitely.
I would prefer no one will set it to None, till change to raise on it.

Reply:
Ahh and this should have been discussed on #122

Reply:
Should be discussed on #122